### PR TITLE
Convert `.stories.mdx` to `stories.js` for compatibility with Storybook v.8

### LINKED ISSUE #4234: Convert Storybook stories to `stories.js` CSF format
## Problem

Storybook 8 has deprecated the combined `stories.mdx` format for stories, replacing them with 2 separate files: `Component.mdx` and `Component.stories.js`. 

## Description

We should convert all `mdx` stories to `.stories.js` CSF format. This will allow us to have fewer changes in the Nuxt 3 migration PR. We can re-add the text documentation from the `stories.mdx` file after migrating to Nuxt 3.
There's an automated script for converting the stories: https://storybook.js.org/docs/migration-guide#storiesmdx-to-mdxcsf

## Alternatives

Keep the stories as is. Remove them in the Nuxt 3 migration PR, and re-add after migrating.

### PR DESCRIPTION
**Pull request by obulat**
## Fixes

Fixes #4234 by @obulat

## Description

This PR converts all the `stories.mdx` to `stories.js`. The text documentation was removed because there is no way in the current setup (the current version of Storybook) to add it. We can re-add it when we move to Nuxt 3 and Storybook 8.
The conversion was done using a Storybook CLI script[^1], and then the errors were fixed manually.

This PR removed the `VSvg` and `VIcon` stories because the import of svg json was non-standard, and it was throwing linting errors.
This PR also removes nuxt/components module that automatically imports components. We were not using this functionality, but it was used by some of the stories.

[^1]: https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#dropping-support-for-storiesmdx-csf-in-mdx-format-and-mdx1-support

## Testing Instructions

Check that the CI works. With the additional smoketest in the base PR, it means that all stories are rendered, with no render error present.

## Checklist

- [ ] My pull request has a descriptive title (not a vague title like`Update index.md`).
- [ ] My pull request targets the _default_ branch of the repository (`main`) or a parent feature branch.
- [ ] My commit messages follow [best practices][best_practices].
- [ ] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [ ] I tried running the project locally and verified that there are no visible errors.
- [ ] I ran the DAG documentation generator (if applicable).

[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: frontend/nuxt.config.ts [hunk 1]
```diff
@@ -280,8 +277,7 @@ const config: NuxtConfig = {
     },
   },
   storybook: {
-    port: 6006, // standard port for Storybook
-    stories: ["~/**/*.stories.@(mdx|js)"],
+    port: 6006,
```
Comment (obulat) [inline frontend/nuxt.config.ts:280]: By default, nuxt storybook scans the `src/components` folder for stories, so we don't need to add this setting here.

File: frontend/src/components/VAudioThumbnail/meta/VAudioThumbnail.stories.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
+import VAudioThumbnail from "~/components/VAudioThumbnail/VAudioThumbnail.vue"
```
Comment (sarayourfriend) [inline frontend/src/components/VAudioThumbnail/meta/VAudioThumbnail.stories.js:1]: It's a shame to add so much untyped and unchecked JavaScript back to the codebase after we spent so much effort converting everything to TypeScript. This was, after all, the main thing I argued against using MDX in the first place for :confused: 

Can you open either a single new issue or multiple issues for us to go through and convert these to TypeScript after the Nuxt 3 migration is finished? Storybook supports it, apparently without any fuss: https://storybook.js.org/docs/writing-stories/typescript

**Review by sarayourfriend — APPROVED** [review]
LGTM! I'm approving based on the existing tests passing (with only changes made to the URLs) and that the smoke test in particular is passing.

File: frontend/src/components/VAudioThumbnail/meta/VAudioThumbnail.stories.js (hunk 2 above)
Comment (obulat) [reply frontend/src/components/VAudioThumbnail/meta/VAudioThumbnail.stories.js:1]: Issue opened: #4290

**Review by AetherUnbound — APPROVED** [review]
I'm not really qualified to give an in-depth review of this, but since CI passes, I'm rubber stamping this so it's not blocked :)

**Pull request merged**